### PR TITLE
Revert Old Message Field Name Changes

### PR DESCRIPTION
**Pull request by gkillough**
These classes will be replaced and will only be used for backward-compatibility with email attachments.

### CODE REVIEW

**Review by bamandel — APPROVED** [review]
It looks good, but why do we need to revert this? Does email use some sort of serialization that requires these names?

**Review by jamesrichard91 — APPROVED** [review]

**Comment by gkillough** [comment]
I'm gonna hold off on merging this until #1360 is merged because I expect some merge conflicts.

**Review by ChomickiM — APPROVED** [review]

**Pull request merged**
